pkgs/by-name/wa/warp-plus/package.nix

```diff
@@ -1,30 +1,32 @@
 {
   lib,
-  buildGoModule,
+  buildGo124Module,
   fetchFromGitHub,
 
   nix-update-script,
   testers,
   warp-plus,
 }:
 
-buildGoModule rec {
+# fails with go 1.25, downgrade to 1.24
+# error tls.ConnectionState: struct field count mismatch: 17 vs 16
+buildGo124Module (finalAttrs: {
   pname = "warp-plus";
-  version = "1.2.6-unstable-2025-09-13";
+  version = "1.2.6-unstable-2025-10-28";
 
   src = fetchFromGitHub {
     owner = "bepass-org";
     repo = "warp-plus";
-    rev = "4af9b2abfc4e79dceea41ac577f1683f62f57b8c";
-    hash = "sha256-7i/06Qn+BRH/bWel9OvgVUAZZSwL2Euv179JDJNn2EE=";
+    rev = "3653f7519d2a08a36222accff6899522bb8b03d0";
+    hash = "sha256-T0YTxQ7iciv5i7lw+bU00B6iYquzBwzYkAlOGZiKeWc=";
   };
 
   vendorHash = "sha256-GmxiQk50iQoH2J/qUVvl9RBz6aIQp8RURqTzrl6NdCY=";
 
   ldflags = [
     "-s"
     "-w"
```

Comment:
I think those 2 flags are no more required.

Reply:
-w is implied by -s, you should drop it.
-s still has an effect over the stdenv stipping, so I'd suggest to leave it there. See https://github.com/NixOS/nixpkgs/issues/346380